src/listbox/ListboxOption.ts

```diff
@@ -14,15 +14,15 @@ export interface ListboxOptionProperties extends ThemedProperties {
 	label: DNode;
 	option: any;
 	selected?: boolean;
-	onClick?(option: any, index: number, key?: string | number): void;
+	onMouseDown?(option: any, index: number, key?: string | number): void;
 }
 
 @theme(css)
 export class ListboxOption extends ThemedMixin(WidgetBase)<ListboxOptionProperties> {
-	private _onClick(event: MouseEvent) {
+	private _onMouseDown(event: MouseEvent) {
 		event.stopPropagation();
-		const { index, key, option, onClick } = this.properties;
-		onClick && onClick(option, index, key);
+		const { index, key, option, onMouseDown } = this.properties;
+		onMouseDown && onMouseDown(option, index, key);
```

Comment:
could this be pointer down?

Reply:
Let me see if that works

Reply:
`onpointerdown` works fine, code is updated 👍